oci/service.go

```diff
@@ -230,6 +232,44 @@ func coreVirtualNetworkService(ctx context.Context, d *plugin.QueryData, region
 	return sess, nil
 }
 
+// DNSService returns the service client for OCI DNS Service
+func DNSService(ctx context.Context, d *plugin.QueryData, region string) (*session, error) {
```

Comment:
Is there any specific reason we are keeping `dns` in caps ?

Reply:
no reason, updated .